p2p/discover/topicindex/topictable.go

```diff
@@ -191,13 +187,12 @@ func (tab *TopicTable) WaitTime(n *enode.Node, t TopicID) time.Duration {
 	regCount := tab.all.Len()
 
 	occupancy := 1.0 - (float64(regCount) / float64(tab.config.AdCacheSize))
-	occupancyScore := float64(tab.config.AdLifetime/time.Second) / math.Pow(occupancy, occupancyExp)
+	occupancyScore := float64(tab.config.AdLifetime/time.Millisecond) / math.Pow(occupancy, occupancyExp)
```

Comment:
What's the intention of this change?

Reply:
Good catch! I actually wasn't sure about that and wanted to ask you. I thought it might be necessary as in Berlin you changed the code to return milliseconds rather than seconds.

Reply:
The field `AdLifetime` has type `time.Duration`, and its unit is nanoseconds. Dividing by `time.Second` here gives the value in seconds. 

But there is a bug here anyway. The current expression truncates the result of the division before converting to float64. It would be more correct to use `AdLifeTime.Seconds()` here to avoid the truncation.